rust/main/chains/hyperlane-sealevel/src/provider.rs

```diff
@@ -68,8 +68,49 @@ impl HyperlaneProvider for SealevelProvider {
         Ok(block_info)
     }
 
-    async fn get_txn_by_hash(&self, _hash: &H512) -> ChainResult<TxnInfo> {
-        todo!() // FIXME
+    async fn get_txn_by_hash(&self, hash: &H512) -> ChainResult<TxnInfo> {
```

Comment:
could add a todo about this call being superfluous but existing for compatibility with the scraper agent abstractions

Reply:
Added.